### PR TITLE
Chrome extn deployment cleanup

### PR DESCRIPTION
**Pull request by muly**

### CODE REVIEW

File: chrome-exten/privacyPolicy.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    <p>No cookies or ad networks are utilized.</p>
+
+    <p>Data collection occurs when user uses our Chrome extension.</p>
+
+    <p>Currently, there is no explicit opt-out option. For concerns, contact us.</p>
+
+    <p>Users can request data deletion or download by contacting us. We are actively working on implementing a process for this.</p>
+
+    <p>Email: <a href="mailto:your@email.com">your@email.com</a></p>
```
Comment (muly) [inline chrome-exten/privacyPolicy.html:54]: need to create and use a real email

File: chrome-exten/privacyPolicy.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+    <p>This Privacy Policy was last updated on [Date].</p>
```
Comment (muly) [inline chrome-exten/privacyPolicy.html:56]: need to replace this with real date

File: chrome-exten/privacy_policy [hunk 3]
```diff
@@ -0,0 +1,24 @@
+
```
Comment (muly) [inline chrome-exten/privacy_policy:1]: need to rename this file or move any required info and delete this file

File: main.go [hunk 4]
```diff
@@ -45,6 +45,9 @@ func main() {
 		http.ServeFile(w, r, "./integration_testing/mock_websites/"+path)
 	})
 
+	router.GET("/privacypolicy.html", func(w http.ResponseWriter, r *http.Request, p httprouter.Params) {
+		http.ServeFile(w, r, "./chrome-exten/privacyPolicy.html")
```
Comment (muly) [inline main.go:49]: lets rename this file to privacy-policy.html

File: main.go [hunk 5]
```diff
@@ -45,6 +45,9 @@ func main() {
 		http.ServeFile(w, r, "./integration_testing/mock_websites/"+path)
 	})
 
+	router.GET("/privacypolicy.html", func(w http.ResponseWriter, r *http.Request, p httprouter.Params) {
```
Comment (muly) [inline main.go:48]: lets rename the path to privacy-policy.html

File: chrome-exten/privacyPolicy.html (hunk 1 above)
Comment (Rohithknaidu) [reply chrome-exten/privacyPolicy.html:54]: okay sir

File: chrome-exten/privacyPolicy.html (hunk 2 above)
Comment (Rohithknaidu) [reply chrome-exten/privacyPolicy.html:56]: okay sir

**Pull request merged**